Dockerfile

```diff
@@ -53,11 +54,15 @@ RUN apt-get update && apt-get install -y \
 	libapparmor-dev \
 	libcap-dev \
 	libltdl-dev \
+	libprotobuf-c0-dev \
+	libprotobuf-dev	\
 	libsqlite3-dev \
 	libsystemd-journal-dev \
 	libtool \
 	mercurial \
 	pkg-config \
+	protobuf-compiler \
+	protobuf-c-compiler \
```

Comment:
I think "yes" because we are building it here, and the dependencies says the compiler is required at build-time: http://criu.org/Installation#Dependencies